src/Aquifer.API/Endpoints/Bibles/Texts/Get/Endpoint.cs

```diff
@@ -51,8 +54,68 @@ public override async Task HandleAsync(Request req, CancellationToken ct)
                         }).ToList()
                 })
                 .ToListAsync(ct);
+            
+            await MapVersificationDifferencesToResponseVersesAsync(req, response, ct);
         }
 
         await (response is null ? SendNotFoundAsync(ct) : SendOkAsync(response, ct));
     }
+
+    private static string FormatBaseBookChapterVerseMapping(int mappedVerseId)
+    {
+        var ( targetBookId, targetChapter, targetVerse ) = BibleUtilities.TranslateVerseId(mappedVerseId);
```

Comment:
Nit: No spaces should be inside the parentheses for tuples.
```suggestion
        var (targetBookId, targetChapter, targetVerse) = BibleUtilities.TranslateVerseId(mappedVerseId);
```